BLSQ/iaso
# IA-931  When deleting a user from the Admin Profile Deactivate a user instead of removing it from DB

Instead of deleting a User I made the change to make use of the is_active field. The user remain but is no longer able to login. As the profile was also deleted, I added a is_active boolean field on the profile ( True by default ). At the moment there is no use of it but in the future ( and step by step I guess ? ) we could use this field to filter profiles that we no longer have to show on differents API. 

By default profile.is_active is True. I changed the Profile Api to accept ?active param which is True by default to ensure that it will not have any impact on the actual configuration and users. But from now, if we deactivate a user the profile will be set as is_active = False and the api using profile wont show the user anymore ( Like on page you will not be able to add a deactivated user for exemple ). If you think about other part of the code where it could make a problem dont hesitate to check it.  

Related JIRA tickets : IA-931

## Self proof reading checklist


## How to test

Delete an account from the profiles api ( user the Admin panel in Iaso ). Check that the user is no longer present in the list. But the user and profile will remain in the django admin. You can also try to login while user is deactivated, you won't be able to login. If you are currently logged and the account is deactivated you will be logged out of Iaso.  


## Notes

We should add a reactivation option for the deactivated users.
